src/tests/unit/test_actions.py

```diff
@@ -11,3 +10,30 @@ def test_get_relay_password(weechat, monkeypatch):
     assert mock_get.call_count == 0
     imp.load_source('get_relay_password', './actions/get-relay-password')
     assert mock_get.call_count == 1
+
+
+def test_update_weeslack(weechat, mock_install_remote,
+                         mock_symlink, mock_chownr, mock_isfile,
```

Comment:
I'm curious, do you have to re-apply these fixtures just to get the Mock object or can you just call the 'called_once_with' directly on the weechat instances which has already applied these fixtures? Something like 'weechat.fetch.install_remote.assert_called_once_with' and remove the fixture on this test?

If that doesn't work well, this is pretty clean too. I like that you added the fixtures to the 'weechat' fixture so you can use it even if you don't need to check the mocks on these specific methods.

Reply:
I've generally found it cleaner to use the mock instances directly where I'm asserting them, and prefer it syntactically. I'm sure addressing it in the above way would work, I just find it messy.